packages/cli/src/eventbus/message-event-bus-destination/message-event-bus-destination-syslog.ee.ts

```diff
@@ -103,6 +103,7 @@ export class MessageEventBusDestinationSyslog
 			);
 		} catch (error) {
 			if (error.message) this.logger.debug(error.message as string);
+			throw error;
```

Comment:
Same Q here

Reply:
Yes this is also caught here: https://github.com/n8n-io/n8n/pull/21448/files#diff-d2065da10e28dde7b5562283ea509115327d708754c95b1c7b2b8473f53c186aR95